app/src/main/java/org/fossasia/phimpme/MyApplication.java

```diff
@@ -36,20 +39,30 @@
 /**
  * Created by dnld on 28/04/16.
  */
+
 public class MyApplication extends Application {
 
     private HandlingAlbums albums = null;
     public static Context applicationContext;
     public ImageLoader imageLoader;
-    private Boolean isPublished = false; // Set this to true at the time of release
-
+    private Boolean isPublished = true; // Set this to true at the time of release
+    private RefWatcher refWatcher;
     public Album getAlbum() {
         return albums.dispAlbums.size() > 0 ? albums.getCurrentAlbum() : Album.getEmptyAlbum();
     }
 
     @Override
     public void onCreate() {
 
+        StrictMode.VmPolicy.Builder builder = new StrictMode.VmPolicy.Builder();
+        StrictMode.setVmPolicy(builder.build());
+
+        if (LeakCanary.isInAnalyzerProcess(this)) {
+            // This process is dedicated to LeakCanary for heap analysis.
+            // You should not init your app in this process.
+            return;
+        }
+        refWatcher = LeakCanary.install(this);
```

Comment:
Leak canary installation not required here. You can check the condition with the help of boolean isPublished and then initialize it.